pages/linux/sam.md

```diff
@@ -0,0 +1,27 @@
+# sam
+
+> AWS Serverless Application Model (SAM) CLI.
+
+- Initialize a serverless application:
+
+`sam init`
+
+- Initialize a serverless application with a specific runtime:
+
+`sam init --runtime python3.7`
```

Comment:
```suggestion
`sam init --runtime {{python3.7}}`
```

Parameters entered by the user should be enclosed in double curly brackets.

Reply:
Updated :+1: